conf/core/ConfigOptions.asciidoc

```diff
@@ -1846,6 +1846,31 @@ If true, InBoundsCriterion requires features exist completely within `in.bounds.
 in order for the criterion to be satisfied. If false, features that cross the bounds may also
 satisfy the criterion.
 
+=== jni.class.path
+
+* Data Type: list
+* Default Value:
+** `${HOOT_HOME}/hoot-josm/target/dependency-jars/josm.jar;${HOOT_HOME}/hoot-josm/target/hoot-josm.jar`
+
```

Comment:
`josm.jar` is a set version of the public josm jar by default and can be changed to whatever jar is needed, but it always gets named the same. `hoot-josm.jar` is the hoot java integration code with josm.